NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj

```diff
@@ -42,15 +42,15 @@
     <ProjectReference Include="..\..\..\WEB\Src\DependencyCollector\DependencyCollector\DependencyCollector.csproj" />
     <ProjectReference Include="..\..\..\WEB\Src\PerformanceCollector\PerformanceCollector\Perf.csproj" />
     <ProjectReference Include="..\..\..\WEB\Src\WindowsServer\WindowsServer\WindowsServer.csproj" />
-    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="1.0.2" />           
+    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="1.1.3" />           
```

Comment:
since we dont support .netcore 1.xx, i think we should update to 2.xx of this

Reply:
v1.1.3 was the last version that supported Net451. 
https://www.nuget.org/packages/Microsoft.AspNetCore.Hosting/1.1.3

starting with v2.0.0 they only support NetStandard 2.0
https://www.nuget.org/packages/Microsoft.AspNetCore.Hosting/2.0.0

I can use a condition to change the version per TargetFramework.
Do you want the lowest v2.0.0 for most compatibility, or the newest v2.2.7?

Reply:
lowest which passes security test. likely 2.1.xx something.

Reply:
https://www.nuget.org/packages/Microsoft.AspNetCore.Hosting/2.1.1

Reply:
Done. :)